Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // help grade 9s calculate the slope of a line given two points
+    String name;
+   System.out.println("Enter value for X1:");
+    // Creates a Scanner used for input
+    Scanner input = new Scanner(System.in);
+     double X1 = input.nextInt();
+    System.out.println("Enter value for Y1:");
+     double Y1 = input.nextInt();
+    System.out.println("Enter value for X2:");
+     double X2 = input.nextInt();
+    System.out.println("Enter value for Y2:");
+     double Y2 = input.nextInt();
+
+      System.out.println("The slope of the line is " +  (Y2 -Y1)/(X2 - X1) );
```

Comment:
Don;t forget the comments in your code. Also remember hat variable names should always begin with a lowercase letter. Finally, try not to do math inside of an output statement. Instead, use a variable to store that value and then use that variable in the output.